float64 latitude_degrees
float64 longitude_degrees